core/src/main/java/apoc/export/json/ImportJson.java

```diff
@@ -35,8 +43,9 @@
 
                     try (final CountingReader reader = FileUtils.readerFor(fileName);
                          final Scanner scanner = new Scanner(reader).useDelimiter("\n|\r");
-                         JsonImporter jsonImporter = new JsonImporter(importJsonConfig, db, reporter)) {
+                         JsonImporter jsonImporter = new JsonImporter(importJsonConfig, db, reporter, tx)) {
                         while (scanner.hasNext()) {
+                            terminationGuard.check();
```

Comment:
```suggestion
                        while (scanner.hasNext() && !Util.transactionIsTerminated(terminationGuard)) {
```

Reply:
Ho creato un `Util.transactionIsTerminated()` con un booleano che ritorna un'eccezione se true perché, non ho ben capito cosa succede sotto banco, ma se non ritorno l'eccezione il test funziona, così come in Neo4j Desktop, ma in quest'ultimo quando faccio la procedura e riavvio mi da un `DBMS failed to start: Error: Could not call remote method 'databaseStatistics'. Check that the method signature is correct. Underlying error: Cannot destructure property 'nodes' of 'e[0]' as it is undefined. ...`